### PR TITLE
Sklearn template

### PR DESCRIPTION
**Pull request by gogasca**

### CODE REVIEW

File: training/templates/sklearn/template/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+cost-effective, and fully managed cloud data warehouse for analytics, with even 
+built-in machine learning.
+
+## Template structure
+```
+template 
+    |__ config
+        |__ config.yaml             # for running normal training job on AI Platform
+        |__ hptuning_config.yaml    # for running hyperparameter tunning job on AI Platform    
+    |__ scripts
```
Comment (happyhuman) [inline training/templates/sklearn/template/README.md]: Our samples (due to modularization) are no longer following this template. We have separated training from prediction. (e.g. we have train-local.sh and train-cloud.sh, as well as many other differences).

File: training/templates/sklearn/template/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+## Using the template
+### Step 0. Prerequisites
+Before you follow the instructions below to adapt the tempate to your machine learning job, 
+you need a Google cloud project if you don't have one. You can find detailed instructions 
+[here](https://cloud.google.com/dataproc/docs/guides/setup-project).
+
+- Make sure the following API & Services are enabled.
+    * Cloud Storage
```
Comment (happyhuman) [inline training/templates/sklearn/template/README.md]: According to our setup page now, this is different from our requirements.

File: training/templates/sklearn/template/README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    * Cloud Machine Learning Engine
+    * BigQuery API
+    * Cloud Build API (for CI/CD integration)
+    * Cloud Source Repositories API (for CI/CD integration)
+
+- Configure project id and bucket id as environment variable.
+  ```bash
+  $ export PROJECT_ID=[your-google-project-id]
+  $ export BUCKET_ID=[your-google-cloud-storage-bucket-name]
```
Comment (happyhuman) [inline training/templates/sklearn/template/README.md]: We are using BUCKET_NAME and not BUCKET_ID

**Comment by gogasca** [comment]
> As I look at this template, I have a few notes to make:
> 
> * This template is now somewhat different from our sklearn sample. I think the differences are enough to make the users confused, since they may not know which one to follow.
> * This template is more about an end-to-end sample. Our samples are now modularized. If we want to provide templates, then we have to break this template into two modular samples, one for training, and one for prediction.
> * At this point, I think our base samples are in a way our "template" for this repo anyway. So, we may not even need to provide such templates at all.

From what I see the existing sklearn example is actually based on this template.
A little bit of background in this template, this is extensively used by customers and PSO team build it. (It provides additional features like Categorical vs Numerical/HPT)
Yes there are minor changes, but seems to be very straightforward to use and I see ours as an implementation.

File: training/templates/sklearn/template/README.md (hunk 1 above)
Comment (gogasca) [reply training/templates/sklearn/template/README.md]: Renamed

**Pull request closed without merging**